### PR TITLE
update from June codebase

### PR DESCRIPTION
**Pull request by kaitejohnson**
Updating with most recent codebase on the development repo. Major changes include:
- additional site-to-state R(t) autoregulation coefficient priors (`autoreg_rt_site_a`, `autoreg_rt_site_b`)
- lots of evaluation functionality
- updates to cmdstan (requires 2.35.0 or greater) + updates to cmdstanr

### CODE REVIEW

File: cfaforecastrenewalww/inst/stan/renewal_ww_hosp_site_level_inf_dynamics
Comment (dylanhmorris) [inline cfaforecastrenewalww/inst/stan/renewal_ww_hosp_site_level_inf_dynamics:1]: looks like a binary was accidentally committed

File: cfaforecastrenewalww/inst/stan/renewal_ww_hosp
Comment (dylanhmorris) [inline cfaforecastrenewalww/inst/stan/renewal_ww_hosp:1]: accidentally committed binary

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
Need to remove accidentally committed binaries.

File: cfaforecastrenewalww/inst/stan/renewal_ww_hosp
Comment (kaitejohnson) [reply cfaforecastrenewalww/inst/stan/renewal_ww_hosp:1]: removed! My bad

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**